src/App.jsx

```diff
@@ -14,70 +16,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {selectedGood.length === 0 ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {`${selectedGood} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setSelectedGood('');
+            }}
+          />
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              className={cn({
+                'has-background-success-light': selectedGood === good,
+              })}
+              key={good}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  data-cy={selectedGood === good ? 'RemoveButton' : 'AddButton'}
+                  type="button"
+                  className={cn('button', {
+                    'is-info': selectedGood === good,
+                  })}
+                  onClick={() => {
+                    if (selectedGood !== good) {
+                      setSelectedGood(good);
+                    } else {
+                      setSelectedGood('');
+                    }
+                  }}
```

Comment:
Same here you can create handler 
```
const onGoodClick = (good) => () => {
  setSelectedGood(prevSeleted => {
    if (prevSeleted === good) {
      return ''
    }
    
    return good;
  })
}
```
usage 
```suggestion
                  onClick={onGoodClick(good)}
```